### PR TITLE
Add handlers for the playbook

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: handlers/handlers.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+- name: restart mesos-slave
+  service:
+    name: mesos-slave
+    enabled: true
+    state: restarted
+
+- name: stop mesos-slave jobs on master
```
Comment (johngian) [inline handlers/handlers.yml]: Please rename this handler to `stop mesos-slave`. The reason why I am suggesting this is:
- To keep it uniform to all handlers
- Here we are not stopping mesos-slave jobs we are disable mesos-slave daemon on masters

**Comment by akatsoulas** [comment]
@johngian pr updated

**Comment by johngian** [comment]
Since we are including `handlers` in all tasks can we just include them once globally?

**Comment by akatsoulas** [comment]
It seems that we need to declare handlers for each host group and since we are including tasks for each host, I don't think that it's possible. This is going to be fixed soon though, via roles.

**Comment by johngian** [comment]
lgtm r+

**Pull request merged**